packages/modules/auth/src/migrations/Migration20241202100304.ts

```diff
@@ -0,0 +1,31 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20241202100304 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table if exists "auth_identity" add column if not exists "deleted_at" timestamptz null;'
+    )
+    this.addSql(
+      'CREATE INDEX IF NOT EXISTS "IDX_auth_identity_deleted_at" ON "auth_identity" (deleted_at) WHERE deleted_at IS NULL;'
+    )
+
+    this.addSql(
+      'alter table if exists "provider_identity" add column if not exists "deleted_at" timestamptz null;'
+    )
+    this.addSql(
+      'CREATE INDEX IF NOT EXISTS "IDX_provider_identity_deleted_at" ON "provider_identity" (deleted_at) WHERE deleted_at IS NULL;'
+    )
```

Comment:
NOTE: The entities were not soft deletable and are now by default, this is why there is this migration.